Mathlib.lean

```diff
@@ -41,6 +41,8 @@ import Mathlib.Lean.Expr.Basic
 import Mathlib.Lean.Expr.ReplaceRec
 import Mathlib.Lean.Expr.Traverse
 import Mathlib.Lean.LocalContext
+import Mathlib.Lean.SourceInfo
+import Mathlib.Lean.mkFreshNameFrom
```

Comment:
Modules should have a capital name. I would put this in `Mathlib.Lean.CoreM`.

Reply:
ok, renamed the file